ohapi/utils_fs.py

```diff
@@ -168,6 +175,186 @@ def load_metadata_csv(input_filepath):
     return metadata
 
 
+def printError(e):
```

Comment:
functions should follow PEP 8 conventions for names, so this would be something like `print_error` – see https://www.python.org/dev/peps/pep-0008/#function-and-variable-names